gapminders/anaconda-project.yml

```diff
@@ -39,7 +38,12 @@ commands:
     notebook: gapminders.ipynb
 
 variables: {}
-downloads: {}
+downloads: 
+  gapminders_data:
+    url: 'https://raw.githubusercontent.com/plotly/datasets/master/gapminderDataFiveYear.csv'
```

Comment:
Let's update this URL not to point to the `master` branch but to a specific commit, so it's more stable.

Reply:
I found the plotly dataset and replaced `master` with the commit id.